ra2ce/analysis/analysis_factory.py

```diff
@@ -34,25 +34,25 @@
 from ra2ce.analysis.direct.analysis_direct_protocol import AnalysisDirectProtocol
 from ra2ce.analysis.direct.direct_damage import DirectDamage
 from ra2ce.analysis.direct.effectiveness_measures import EffectivenessMeasures
-from ra2ce.analysis.indirect.analysis_indirect_protocol import AnalysisIndirectProtocol
-from ra2ce.analysis.indirect.losses import Losses
-from ra2ce.analysis.indirect.multi_link_isolated_locations import (
+from ra2ce.analysis.losses.analysis_losses_protocol import AnalysisLossesProtocol
+from ra2ce.analysis.losses.losses import Losses
```

Comment:
why do we have this import? This needs to be fixed asap. Keep in mind it would bring more strength to the argument of having `Losses` (previously as `Indirect` ) as `Loss`